App.js

```diff
@@ -1,48 +1,114 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, Image, Fonts, Button } from 'react-native';
+import { Font } from 'expo';
 
 export default class App extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./assets/fonts/Double_Bubble_shadow.otf'),
+    });
+    this.setState({ fontLoaded: true });
+  }
   render() {
     return (
       <View style={styles.container}>
-        <View style={{flexDirection: 'row', paddingTop: 30}}>
-          <View>
-            <View style={styles.pongRoundedRectangle}>
-              <Text style={styles.pongMatesText}>Pong</Text>
+        <View style = {{flexDirection: 'row', justifyContent: 'space-around'}}>
+          <View style = {styles.topButtons}>
+            <Image source = {require('./assets/profile.png')} style = {styles.image}/>
+          </View>
+          <View style = {styles.topButtons}>
+            <Text style = {styles.faqButton}>?</Text>
+          </View>
+        </View>
+
+        <View style = {{justifyContent: 'center'}}>
+
+          <View style = {{width: '100%', flexDirection: 'row', justifyContent: 'flex-start'}}>
+            
+            <View style = {styles.postButton}>
+            {
+              this.state.fontLoaded ? (
+                <Text style = {styles.postFontStyle}>Create a Game</Text>
+              ) : null
```

Comment:
What happens if `this.state.fontLoaded` is `false`? There should be text there whether or not the font loads properly